mhkit/tests/utils/test_utils.py

```diff
@@ -29,10 +29,10 @@ def test_get_statistics(self):
         # load in file
         df = self.data["loads"]
         df.Timestamp = pd.to_datetime(df.Timestamp)
-        df.set_index("Timestamp", inplace=True)
+        test_df = df.set_index("Timestamp")
```

Comment:
inplace was overwriting the original dataset and prevented self.data["loads"] to be resued consistently in new tests e.g. `def test__calculate_statistics(self):`